apps/web/src/components/templates/email-editor/TestSendEmail.tsx

```diff
@@ -94,6 +94,7 @@ export function TestSendEmail({ index, isIntegrationActive }: { index: number; i
 
             return query;
           }}
+          data-test-id="test-email-emails"
```

Comment:
```suggestion
          data-test-id="test-send-email"
```
From the file name though maybe I am missing some functionality here.

Reply:
this is an input field for emails the button to send have this test-id...